aiohttp/web_ws.py

```diff
@@ -162,9 +162,8 @@ def _send_heartbeat(self) -> None:
     def _pong_not_received(self) -> None:
         if self._req is not None and self._req.transport is not None:
             self._closed = True
-            self._close_code = WSCloseCode.ABNORMAL_CLOSURE
+            self._set_code_close_transport(WSCloseCode.ABNORMAL_CLOSURE)
```

Comment:
Having trouble writing a test for this but that's probably my lack of sleep

We could leave this as is since it's not covered by existing tests.

Will keep trying though